frontend/src/metabase/admin/performance/strategies.ts

```diff
@@ -33,9 +33,13 @@ export const doNotCacheStrategyValidationSchema = Yup.object({
   type: Yup.string().equals(["nocache"]),
 });
 
+export const defaultMinDurationMs = 1000;
 export const ttlStrategyValidationSchema = Yup.object({
   type: Yup.string().equals(["ttl"]),
-  min_duration_ms: positiveInteger.default(60000),
+  min_duration_ms: positiveInteger.default(defaultMinDurationMs),
+  min_duration_seconds: positiveInteger.default(
+    Math.ceil(defaultMinDurationMs / 1000),
+  ),
```

Comment:
I'm not a huge fan of the complexity of having both _ms and _seconds properties in the validation schema here. I understand why you're doing it this way, but I wonder if it might be less problematic in a separate-forms-for-separate-strategies situation where all form submission logic doesn't have to go through a `translateConfig` function to handle it.

I think this could be a lot clearer why/how this is happening if the conversion happens when data goes into/out of the specific form it applies to.